include/vara/Feature/FeatureSourceRange.h

```diff
@@ -72,70 +72,72 @@ class FeatureSourceRange {
   };
 
   class FeatureMemberOffset {
-    FeatureMemberOffset(std::string MemberOffset)
-        : MemberOffset(std::move(MemberOffset)) {}
-
   public:
-    static llvm::Optional<FeatureMemberOffset>
-    createFeatureMemberOffset(std::string MemberOffset) {
-      // Member offsets need to have excaptly one :: separator
-      if (isInMemberOffsetFormat(MemberOffset)) {
-        return FeatureMemberOffset(std::move(MemberOffset));
+    static std::optional<FeatureMemberOffset>
+    createFeatureMemberOffset(llvm::StringRef MemberOffset) {
```

Comment:
Please put the "splitting" semantics in a helper function, so we have a better abstraction.